Docs: components, get started pages update (#490)
exadel-inc/ui-playground: src/plugins/header/README.md

# UIP Header

**UIPHeader** - custom element, a container for [UIPSnippets](src/header/snippets/README.md) and [UIPOptions](src/header/options/README.md).

# Description

> By default *UIPHeader* initializes *UIPSnippets*, *UIPOptions* and a copy icon inside of it, so you need to add an empty *uip-header* tag only. But you can also manually set what stuff do you want it to render. The following example shows how you can display [UIPSnippets](src/header/snippets/README.md) only:

Reviewer: ```suggestion
By default *UIPHeader* initializes *UIPSnippets*, *UIPOptions*, and a copy icon inside of it, so you need to add an empty *uip-header* tag only. But you can also manually set what you want to render. The following example shows how only [UIPSnippets](src/header/snippets/README.md) can be displayed:
```
